data/json/monsters/mammal.json

```
@@ -197,7 +197,9 @@
     "upgrades": { "half_life": 13, "into_group": "GROUP_RATKIN_EVOLVED" },
     "fear_triggers": [ "PLAYER_CLOSE", "SOUND" ],
     "special_attacks": [ [ "EAT_FOOD", 120 ] ],
-    "flags": [ "SEES", "SMELLS", "HEARS", "KEENNOSE", "WARM", "SWIMS", "ANIMAL", "PATH_AVOID_DANGER_1" ]
+    "petfood": { "food": [ "DOGFOOD" ], "feed": "The %s seems to like you!  It lets you pat its head and seems friendly." },
+    "pet": "The %s lets out a happy squeak as you give its head a small scritch.",
+    "flags": [ "SEES", "SMELLS", "HEARS", "KEENNOSE", "CANPLAY", "WARM", "SWIMS", "ANIMAL", "PATH_AVOID_DANGER_1" ]
```

```suggestion
    "petfood": {
      "food": [ "DOGFOOD" ],
      "feed": "The %s seems to like you!  It lets you pat its head and seems friendly.",
      "pet": "The %s lets out a happy squeak as you give its head a small scritch."
    },
    "flags": [ "SEES", "SMELLS", "HEARS", "KEENNOSE", "CANPLAY", "WARM", "SWIMS", "ANIMAL", "PATH_AVOID_DANGER_1" ]
```

Just a small correction, moving the `"pet"` field inside the `"petfood"` object. Tested in-game and it works (hopefully this resolves the stalled test).